Plugins/Source/KISAddonPointer.cs

```diff
@@ -12,8 +12,8 @@ namespace KIS {
 
 [KSPAddon(KSPAddon.Startup.Flight, false)]
 sealed class KISAddonPointer : MonoBehaviour {
-  public GameObject audioGo = new GameObject();
-  public AudioSource audioBipWrong = new AudioSource();
+  public GameObject audioGo = null;
+  public AudioSource audioBipWrong = null; //new AudioSource();
```

Comment:
I left it as there was a double-init for audioBipWrong : the initializer was "new AudioSource()" (which raises an error), and Awake() already had the current code for this member. So I'm not sure why that field was being initialized twice; figured the maintainer could then make an executive decision on this.